src/flycapture2_enums.pxi

```diff
@@ -268,3 +268,30 @@ TIFF_CCITTFAX3 = FC2_TIFF_CCITTFAX3
 TIFF_CCITTFAX4 = FC2_TIFF_CCITTFAX4
 TIFF_LZW = FC2_TIFF_LZW
 TIFF_JPEG = FC2_TIFF_JPEG
+
```

Comment:
This file is automatically generated. See the comment at the beginning. Please don't add manual stuff here.
I'd move the dict to `flycapture2.pyx`